.travis.yml

```diff
@@ -5,8 +5,9 @@ import: scala/scala-dev:travis/default.yml
 language: scala
 
 scala:
-  - 2.12.11
-  - 2.13.2
+  # Add when released
+  # - 2.12.12 when
```

Comment:
I'd suggest opening an issue to track this.

Reply:
https://github.com/scala/scala-async/issues/241